doc/source/examples/networktheory/CPW media example.ipynb

```diff
@@ -0,0 +1,271 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CPW media example\n",
+    "\n",
+    "A coplanar waveguide (CPW) is a planar transmission line where the return path is a plane located on both sides of a strip conductor on the same layer. The characteristic impedance depends on the strip width and the gap with the return plane, giving two degrees of freedom to the designer.\n",
+    "\n",
+    "A great advantage of CPW is the convenience to perform shunts or series connections with components because both the signal and the return path are on the same side, thus no via is required. On the other hand, the large return planes consume a lot of surfaces on the circuit.\n",
```

Comment:
"the large return planes consume a lot of surfaces on the circuit." should be reworded. For example "the large return planes require a lot of area."

Reply:
On the other hand, the large coplanar planes require a lot of area.